CDCgov/cdc-open-viz
# fix [DEV-9652] legend border off center

## [DEV-9652](https://websupport-cdc.msappproxy.net/browse/DEV-9652)

## Testing Steps

1. Open or create map with linear block gradient legend
2. Observe there is now an even border surrounding the color blocks

## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)
### Problem:
<img width="1075" alt="Screenshot 2024-10-28 at 2 49 24 PM" src="https://github.com/user-attachments/assets/3f1b14ea-95d3-4773-88e7-50f6ea578e99">

### Solution
<img width="1045" alt="Screenshot 2024-10-28 at 2 50 06 PM" src="https://github.com/user-attachments/assets/dc5341e2-9881-494a-98ba-83fd2aede228">
